Dockerfile

```diff
@@ -6,9 +6,8 @@ LABEL maintainer="Renoki Co. <alex@renoki.org>"
 
 COPY . /app
 
-RUN npm install typescript@latest -g && \
+RUN npm install typescript@latest pm2 -g && \
```

Comment:
pm2 is installed, but will it run with it (the ENTRYPOINT)?
And also, maybe best use `pm2@latest`

Reply:
it will not run with entrypoint, but that is probably not needed, it is there just to add the option to use it for people that want to

Reply:
alright noted.